.github/workflows/helm_charts_scalar.yml

```diff
@@ -41,31 +41,31 @@ jobs:
           helm repo add scalar https://scalar-labs.github.io/helm-charts
 
       - name: Set up chart-testing
-        uses: helm/chart-testing-action@v2.0.1
+        uses: helm/chart-testing-action@v2
```

Comment:
I don't specify the specific patch version to use the latest stable version in the CI.